spec/serializers/user_serializer_spec.rb

```diff
@@ -0,0 +1,161 @@
+require 'rails_helper'
+
+describe 'UserSerializer', type: :serializer do
+  describe 'instance methods' do
+    describe '#serializable_hash' do
+      context 'when I provide a valid user' do
+        let(:user) { create(:user) }
+        let(:user_hash) { UserSerializer.new(user).serializable_hash }
+        let(:user_data) { user_hash[:data] }
+        let(:user_attributes) { user_data[:attributes] }
+
+        it 'formats the single user response for delivery', :aggregate_failures do
+          expect(user_hash).to be_a Hash
+          expect(user_hash.size).to eq(1)
+
+          expect(user_hash).to have_key(:data)
+
+          expect(user_data).to be_a Hash
+          expect(user_data.size).to eq(3)
+
+          expect(user_data).to have_key(:id)
+          expect(user_data[:id]).to be_a String
+
+          expect(user_data).to have_key(:type)
+          expect(user_data[:type]).to be_a Symbol
+          expect(user_data[:type]).to eq(:user)
+
+          expect(user_data).to have_key(:attributes)
+          expect(user_attributes).to be_a Hash
+          expect(user_attributes.size).to eq(13)
+
+          expect(user_attributes).to have_key(:email)
+          expect(user_attributes[:email]).to be_a String
+
+          expect(user_attributes).to have_key(:google_id)
+          expect(user_attributes[:google_id]).to be_a String
+
+          expect(user_attributes).to have_key(:google_token)
+          expect(user_attributes[:google_token]).to be_a String
+
+          expect(user_attributes).to have_key(:google_image_url)
+          expect(user_attributes[:google_image_url]).to be_a String
+
+          expect(user_attributes).to have_key(:zip_code)
+          expect(user_attributes[:zip_code]).to be_a String
+
+          expect(user_attributes).to have_key(:summary)
+          expect(user_attributes[:summary]).to be_a String
+
+          expect(user_attributes).to have_key(:goal)
+          expect(user_attributes[:goal]).to be_an Integer
+
+          expect(user_attributes).to have_key(:availability_morning)
+          expect(user_attributes[:availability_morning]).to be_in([true, false])
+
+          expect(user_attributes).to have_key(:availability_afternoon)
+          expect(user_attributes[:availability_afternoon]).to be_in([true, false])
+
+          expect(user_attributes).to have_key(:availability_evening)
+          expect(user_attributes[:availability_evening]).to be_in([true, false])
+
+          expect(user_attributes).to have_key(:first_name)
+          expect(user_attributes[:first_name]).to be_a String
+
+          expect(user_attributes).to have_key(:last_name)
+          expect(user_attributes[:last_name]).to be_a String
+
+          expect(user_attributes).to have_key(:full_name)
+          expect(user_attributes[:full_name]).to be_a String
+
+          # TODO: Add relationships for followers and followees?  Or will we do
+          #       that in a separate serialized hash?
```

Comment:
We need to discuss as a group how we need to send `follower` and `followee` data through the endpoint and how that gets handled by the front end.